src/components/comment-form/comment-form.jsx

```diff
@@ -0,0 +1,66 @@
+import React, {useState} from 'react';
+
+const CommentForm = () => {
+  const [comment, setComment] = useState({
```

Comment:
Предлагаю для единообразия, чтобы в любом месте сразу было понятно, о чём идёт речь, именовать так:
```suggestion
  const [state, setState] = useState({
```
Но это как тебе удобней)